Src/Microsoft.Dynamic/Actions/ReflectedPropertyTracker.cs

```diff
@@ -80,11 +74,7 @@ public class ReflectedPropertyTracker : PropertyTracker {
             return _propInfo.GetIndexParameters();
         }
 
-        public PropertyInfo Property {
-            get {
-                return _propInfo;
-            }
-        }
+        public PropertyInfo Property => _propInfo;
```

Comment:
I'm not seeing any extra lines because of a property inside a property. Only differences I'm seeing in the IL are the `CompilerGeneratedAttribute` and the field access getting changed to property calls (which doesn't change anything because the JIT will inline them anyway).

In any case I'm fine with the PR as it is, I was just curious about the reasoning...